graphix/sim/statevec.py

```diff
@@ -8,9 +8,9 @@
 class StatevectorBackend:
     """MBQC simulator with statevector method."""
 
-    def __init__(self, pattern, max_qubit_num=20):
+    def __init__(self, pattern, max_qubit_num=20, pr_calc=False):
```

Comment:
Is there a reason to set this at the instantiation of the backend? 
how about adding `pr_calc` as kwarg for `measure()`?

Reply:
I probably see the intension now, is it so that we can pass the option directly from `pattern.simulate_pattern` kwarg? If so I think it makes perfect sense!

Reply:
Sorry for the late reply. Yes that was the intention :).